app/common/services/endpoints/form-stages.js

```diff
@@ -10,7 +10,7 @@ function (
         formId: '@formId',
         id: '@id',
         order: 'asc',
-        orderby: 'priority'
+        orderby: '@priority'
```

Comment:
Adding an @ here will try and pull `priority` from the params passed to this endpoint.. We don't want that, we just always want to fetch stages sorted by priority.